### ISSUE TITLE
Croparia v 1.18 not loading with Forge v 14.23.4.2705 or above...

### ISSUE THREAD

**Issue by TheOrionsun** [PowerInMC/Croparia#6]
Hello,

I am not able to load a MC 1.12.2- Forge game above v 14.23.4.2704. I have 63 mods and only this mod crashes the MC load. Removing it and all works fine and staying at Forge 2704 and it loads fine.

Any thoughts?

Thank you for the mod!

**Comment by forestbat** [comment]
Please give me your crash-report。

**Comment by TheOrionsun** [comment]
Hello,

 

I have been testing different mods with Croparia and it looks to have load issues with several mods.

 

I use Curse and created a new custom profile with Croparia v 1.18, MC 1.12.2, Forge v .2725 and Industrial Craft-2-2.8.73.

 

The attached crash txt is what I get when just these two mods are loaded. 

 

If I load Industrial Craft-2-2.8.16-ex112 (just using release versions) they do not cause a crash.

 

Not sure what is going on, but I have tried other mods one at a time and MC will not load.

 

I noticed the crashes started when Forge moved to v 2705. From there I cannot load Croparia with other mods I use.

 

The crash report 06.59.16 (using forge v 2705) and crash report 06.40.44 (using forge v 2725).

 

Hopes this helps…

 

Thank you for your time,

 

Dan
